security/azure/restrict_rdp_internet/azure_restrict_rdp_inet.pt

```diff
@@ -0,0 +1,203 @@
+name "Azure Restrict RDP from Internet"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that sends email notifications when an Azure Network Security Group has RDP open to the internet. See the [README](https://github.com/flexera/policy_templates/tree/master/security/azure/restrict_rdp_internet) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
```

Comment:
Replace "A policy that sends email notifications" with "Reports"

Mentioning that it's a policy is redundant, and the policy will also raise an incident within Flexera One, so it doesn't *just* send email notifications.

Reply:
Great call out, changed.